src/actions/arcActions.ts

```diff
@@ -1100,6 +1090,8 @@ export function redeemProposal(daoAvatarAddress: string, proposal: IProposalStat
     try {
       const redeemerInstance = await Arc.RedeemerFactory.deployed();
       const redeemTx = await redeemerInstance.redeem({ avatarAddress: daoAvatarAddress, beneficiaryAddress: accountAddress, proposalId: proposal.proposalId });
+      await redeemTx.getTxConfirmed();
```

Comment:
hmm, this does not seem like a good idea to me. don't we need to wait for the RedeemerRedeem event to fire off the onRedeemEvent? this wont work if they refresh the page right?

Reply:
The `RedeemerRedeem` event was removed, no?

Reply:
oh boy, yeah forgot to mention that we actually could still use it for this case. what do you suggest we do?

Reply:
Well, we can use the `operations` module that gets persisted through refreshes.

Reply:
https://github.com/daostack/alchemy/pull/155/commits/50a3d168b968097e2b386c2b0639fd4956c64f02